packages/sitecore-jss/src/dictionary-service.ts

```diff
@@ -0,0 +1,85 @@
+import { AxiosResponse } from 'axios';
+import mcache from 'memory-cache';
+import { AxiosDataFetcher } from './data-fetcher';
+import { DictionaryPhrases, DictionaryServiceData } from './dataModels';
+import { HttpJsonFetcher } from './httpClientInterface';
+
+export type DictionaryServiceConfig = {
+  apiHost: string;
```

Comment:
Should add JSDoc for these 3 params (looks like layout-service is missing these JSDoc comments as well).

Reply:
Added